android/KMAPro/kMAPro/src/main/AndroidManifest.xml

```diff
@@ -40,12 +47,35 @@
                 <action android:name="android.intent.action.MAIN" />
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
+
             <intent-filter>
                 <action android:name="android.intent.action.VIEW" />
                 <category android:name="android.intent.category.DEFAULT" />
                 <category android:name="android.intent.category.BROWSABLE" />
-                <data android:scheme="keyman" android:host="localhost" android:path="/open"/>
+
+                <!-- Intending to deprecate keyman:// protocol in https://github.com/keymanapp/keyman/issues/538 -->
+                <data android:scheme="keyman" android:host="localhost" android:path="/open" />
+            </intent-filter>
+
+            <intent-filter>
+                <action android:name="android.intent.action.VIEW" />
+                <category android:name="android.intent.category.DEFAULT" />
+                <category android:name="android.intent.category.BROWSABLE" />
+
+                <!-- local kmp file -->
+                <data android:scheme="file" android:host="*" android:mimeType="*/*" android:pathPattern=".*\\.kmp" />
```

Comment:
Does the `pathPattern` match the whole string or do we need a `$` on the end of it?

Reply:
I've tested with a local `.kmpo` file and verified that doesn't trigger the Keyman intent.

We shouldn't need a `$`, and I haven't seen anybody using it in my searches for `pathPattern`

Reply:
mea culpa. It turns out a `.kmpo` file does trigger the Keyman app. Unfortunately, Android uses [PatternSimpleGlob](https://developer.android.com/reference/android/os/PatternMatcher.html#PATTERN_SIMPLE_GLOB) for the PatternMatcher of pathPattern, and characters like `$` are not available.

A follow-on PR will have the Keyman app display a notification when it's ignoring non-kmp files.